packages/devtools_app/lib/src/service/editor/api_classes.dart

```diff
@@ -396,3 +425,34 @@ class EditorDevice with Serializable {
         Field.supported: supported,
       };
 }
+
+/// UI settings for an editor's theme.
+class EditorTheme with Serializable {
+  EditorTheme({
+    required this.isDarkMode,
+    required this.backgroundColor,
+    this.foregroundColor,
```

Comment:
Any reason `foregroundColor` is not required but others are?

Reply:
Changing back; I think I changed this for an outdated test